bazel/external/gcovr.BUILD

```diff
@@ -1,5 +1,15 @@
-sh_binary(
+load("@subpar//:subpar.bzl", "par_binary")
+
+par_binary(
     name = "gcovr",
-    srcs = ["scripts/gcovr"],
+    srcs = [":renamed_gcovr.py"],
+    main = ":renamed_gcovr.py",
     visibility = ["//visibility:public"],
 )
+
+genrule(
```

Comment:
Please add a comment explaining why this is necessary. I think we used to have one problem with gcovr, and now we have two (gcovr + Bazel), but it seems you're mostly there.

Reply:
Done.